src/pages/SponsorTheTeam.js

```diff
@@ -21,7 +21,7 @@ export default function SponsorTheTeam() {
   const handleSubmit = async (e) => {
     e.preventDefault();
     const data = {
-      access_key: '660265ba-10e3-4c70-a956-ad48ae8857c6',
+      access_key: '30522979-edc6-4332-a439-8e80341df0f3',
```

Comment:
Is it safe to have this key publicly viewable?

Reply:
Cool.  Just wanted to be sure this wasn't going to cause trouble and it looks like it won't